exe-unit/components/gsb-http-proxy/src/gsb_to_http.rs

```diff
@@ -76,35 +90,64 @@ impl GsbToHttpProxy {
 
         Box::pin(stream)
     }
+
+    pub fn requests_counter(&mut self) -> impl Counter {
+        self.counters.requests_counter()
+    }
+
+    pub fn requests_duration_counter(&mut self) -> impl Counter {
+        self.counters.requests_duration_counter()
+    }
```

Comment:
Counters are exposed only as some `impl Counter` Instead of `Monitor` trait implementations.